include/crow/http_server.h

```diff
@@ -213,6 +224,7 @@ namespace crow
         std::vector<std::unique_ptr<asio::io_service>> io_service_pool_;
         std::vector<detail::task_timer*> task_timer_pool_;
         std::vector<std::function<std::string()>> get_cached_date_str_pool_;
+        std::vector<std::atomic<uint>> task_queue_length_pool_;
```

Comment:
Please replace this with `unsigned int` or `unsigned` (see #54 for more details).

Reply:
Done! Didn't know about that.